### ISSUE TITLE
How to get the pre-train model of VGG19？

### ISSUE THREAD

**Issue by lssily** [facebookresearch/Detectron#672]
Hello！I want to implement Mask R-CNN on VGG19_FPN backbone. However, I didn't find the pre-train model of VGG19. Look forward to your answer!

**Comment by ir413** [comment]
Hi @lssily, unfortunately we do not have any pretrained models with a VGG19 backbone available at this time. Sorry for the inconvenience.
